resources/resources.yaml

```diff
@@ -97,21 +97,16 @@ spec:
         - name: log-forwarder
           image: log-forwarder:latest
           volumeMounts:
-            - name: gravitylog
-              mountPath: /var/log/gravity
-            - name: varlogcontainers
-              mountPath: /var/log/containers
+            - name: varlog
+              mountPath: /var/log
             - name: extdockercontainers
               mountPath: /ext/docker/containers
             - name: gravitysite
               mountPath: /var/lib/gravity/site
       volumes:
-        - name: gravitylog
-          hostPath:
-            path: /var/log/gravity
-        - name: varlogcontainers
+        - name: varlog
           hostPath:
-            path: /var/log/containers
+            path: /var/log
```

Comment:
I remember I was having an issue in monitoring app when I mounted the whole `/etc` into influxdb container (its config was in `/etc/influxdb.conf`). The issue was that during update the old pod couldn't exit because for some reason it apparently tried to wipe the whole /etc during pod teardown (no idea why) and getting "/etc/resolv.conf is busy" or somtething like this. Not sure it applies here but just wanted to mention it. Issues may arise later when upgrading from _this_ version.

Reply:
I don't think there's any harm in exposing entire `/var/log` (as there was in your case) - although Pod trying to remove the contents of the mounted directory is suspicious.